src/components/instant-buy/success.tsx

```diff
@@ -0,0 +1,36 @@
+"use client"
+import { WarningOctagon } from "@phosphor-icons/react"
+import React from "react"
+
+import { Button } from ".."
+
+interface Props {
+	txnHash: string
+}
+
+const Success = (props: Props) => {
+	const viewTxn = () => {
```

Comment:
We don't need this function. wrap it in a link that opens a new tab. 
No need to call window